### PR TITLE
Add Steve Rydz

### PR DESCRIPTION
**Pull request by steverydz**

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -11382,6 +11382,16 @@ module.exports = [
       'GraphQL',
     ],
   },
+  {
+    name: 'Steve Rydz',
+    description: 'Developer',
+    url: 'https://steverydz.com/uses',
+    twitter: '@steverydz',
+    country: 'United Kingdom',
```
Comment (gatarelib) [inline src/data.js]: "country" must be one of [🇦🇫, 🇦🇱, 🇩🇿, 🇦🇸, 🇦🇩, 🇦🇴, 🇦🇮, 🇦🇶, 🇦🇬, 🇦🇷, 🇦🇲, 🇦🇼, 🇦🇺, 🇦🇹, 🇦🇿, 🇧🇮, 🇧🇸, 🇧🇭, 🇧🇩, 🇧🇧, 🇧🇾, 🇧🇪, 🇧🇿, 🇧🇯, 🇧🇲, 🇧🇹, 🇧🇴, 🇧🇶, 🇧🇦, 🇧🇼, 🇧🇻, 🇧🇷, 🇧🇳, 🇧🇬, 🇧🇫, 🇰🇭, 🇨🇲, 🇨🇦, 🇨🇻, 🇰🇾, 🇨🇫, 🇹🇩, 🇨🇱, 🇨🇳, 🇨🇽, 🇨🇨, 🇨🇴, 🇰🇲, 🇨🇩, 🇨🇬, 🇨🇰, 🇨🇷, 🇭🇷, 🇨🇺, 🇨🇼, 🇨🇾, 🇨🇿, 🇩🇰, 🇩🇯, 🇩🇲, 🇩🇴, 🇪🇨, 🇪🇬, 🇸🇻, 🇬🇶, 🇪🇷, 🇪🇪, 🇪🇹, 🇫🇴, 🇫🇯, 🇫🇮, 🇫🇷, 🇬🇫, 🇵🇫, 🇹🇫, 🇬🇦, 🇬🇲, 🇬🇪, 🇩🇪, 🇬🇭, 🇬🇮, 🇬🇧, 🇬🇷, 🇬🇱, 🇬🇩, 🇬🇵, 🇬🇺, 🇬🇹, 🇬🇬, 🇬🇳, 🇬🇼, 🇬🇾, 🇭🇹, 🇭🇳, 🇭🇰, 🇭🇺, 🇮🇸, 🇮🇳, 🇮🇩, 🇮🇷, 🇮🇶, 🇮🇪, 🇮🇱, 🇮🇹, 🇨🇮, 🇯🇲, 🇯🇵, 🇯🇪, 🇯🇴, 🇰🇿, 🇰🇪, 🇰🇮, 🇰🇼, 🇰🇬, 🇱🇦, 🇱🇻, 🇱🇧, 🇱🇸, 🇱🇷, 🇱🇾, 🇱🇮, 🇱🇹, 🇱🇺, 🇲🇴, 🇲🇰, 🇲🇬, 🇲🇼, 🇲🇾, 🇲🇻, 🇲🇱, 🇲🇹, 🇲🇭, 🇲🇶, 🇲🇷, 🇲🇺, 🇾🇹, 🇲🇽, 🇫🇲, 🇲🇩, 🇲🇨, 🇲🇳, 🇲🇪, 🇲🇸, 🇲🇦, 🇲🇿, 🇲🇲, 🇳🇦, 🇳🇷, 🇳🇵, 🇳🇱, 🇳🇨, 🇳🇿, 🇳🇮, 🇳🇪, 🇳🇬, 🇳🇺, 🇳🇫, 🇰🇵, 🇲🇵, 🇳🇴, 🇴🇲, 🇵🇰, 🇵🇼, 🇵🇸, 🇵🇦, 🇵🇬, 🇵🇾, 🇵🇪, 🇵🇭, 🇵🇳, 🇵🇱, 🇵🇹, 🇵🇷, 🇶🇦, 🇷🇪, 🇷🇴, 🇷🇺, 🇷🇼, 🇼🇸, 🇸🇲, 🇸🇹, 🇸🇦, 🇸🇳, 🇷🇸, 🇸🇨, 🇸🇱, 🇸🇬, 🇸🇰, 🇸🇮, 🇸🇧, 🇸🇴, 🇿🇦, 🇬🇸, 🇰🇷, 🇸🇸, 🇪🇸, 🇱🇰, 🇸🇩, 🇸🇷, 🇸🇯, 🇸🇿, 🇸🇪, 🇨🇭, 🇸🇾, 🇹🇼, 🇹🇯, 🇹🇿, 🇹🇭, 🇹🇱, 🇹🇬, 🇹🇰, 🇹🇴, 🇹🇹, 🇹🇳, 🇹🇷, 🇹🇲, 🇹🇨, 🇹🇻, 🇺🇬, 🇺🇦, 🇦🇪, 🇺🇾, 🇺🇸, 🇺🇿, 🇻🇺, 🇻🇦, 🇻🇪, 🇻🇳, 🇼🇫, 🇪🇭, 🇾🇪, 🇿🇲, 🇿🇼, 🏴󠁧󠁢󠁥󠁮󠁧󠁿, 🏳️‍🌈, 🇪🇺, 🏴󠁧󠁢󠁳󠁣󠁴󠁿]

**Review by gatarelib — CHANGES_REQUESTED** [review]

File: src/data.js (hunk 1 above)
Comment (steverydz) [reply src/data.js]: Thanks @gatarelib. Updated

File: readme.md [hunk 2]
```diff
@@ -170,6 +170,7 @@ This readme is auto-generated from the data.js file, so please don't PR this fil
 * [Andrew Byrd](https://www.andrewbyrd.dev/uses/) — Web dev. Girl Dad. Bodybuilder. Gamer.
 * [Siim Männart](https://siim.me/uses) — Cloud Engineer and Developer. Learner of things, recreational pilot and lover of cats. From Tallinn, Estonia. Based in Sydney, Australia.
 * [Prakhil TP](https://www.notion.so/Things-Prakhil-uses-e995e61834c242f1b739be9f8819fb0c) — Team lead, Experienced full-stack engineer & non-stop learner.  :wq
+* [Steve Rydz](https://steverydz.com/uses) — Linux user, Open source, Web developer
```
Comment (HugoDF) [inline readme.md:173]: This files gets updated automatically, probably best to remove this change

Comment (steverydz) [reply readme.md]: @HugoDF sorry, updated, thanks

**Pull request merged**